django_elasticsearch_dsl/management/commands/search_index.py

```diff
@@ -63,6 +65,18 @@ def add_arguments(self, parser):
             dest='parallel',
             help='Run populate/rebuild update single threaded'
         )
+        parser.add_argument(
+            '--atomic',
```

Comment:
The naming of the flags look a little bit confusing to me, maybe we could use something like `--use-alias`, what do you think?

Reply:
`--use-alias` works for me - overall my naming on the flags was something I wasn't to set on. Similarly, I think `--atomic-no-delete` could/should now be `--use-alias-no-delete-indices`.

At the moment, the default behavior of `--use-alias` is to delete the old indices. Do you think that is fine in being the default, or should the deletion of the old indices have to be explicit via another flag? I.e. get rid of `--use-alias-no-delete-indices` and add something like `--use-alias-delete-indices`.

Reply:
@safwanrahman What are your thoughts on this?

Reply:
@saadmk11 @safwanrahman Just wanted to follow up - what are your thoughts?

Reply:
@oehrlein Sorry for not getting back to it. I was quite busy with some issues. I think `--use-alias` is better suited for the term. I think the deletion should be by default, we can add another flag that will not delete the old index. What do you think?

Reply:
@safwanrahman No worries, I was busy over the past week as well! 

I agree that `--use-alias` is the best choice and went ahead and made the changes. Likewise, I kept deletion as the default behavior and renamed `--atomic-no-delete` to `--use-alias-keep-index`.